### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maninhill**
更新 README。

#### Please indicate you've done the following:

- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Made sure commit message follow the rule of [Conventional Commits specification](https://www.conventionalcommits.org/).
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed.

### CODE REVIEW

**Review by liuruibin — DISMISSED** [review]
/approve
/lgtm

File: README.md [hunk 1]
```diff
@@ -9,20 +9,22 @@
...
+MeterSphere 是一站式开源持续测试平台, 涵盖测试跟踪、接口测试、UI 测试和性能测试等功能，全面兼容 JMeter、Selenium 等主流开源标准，有效助力开发和测试团队充分利用云弹性进行高度可扩展的自动化测试，加速高质量的软件交付，推动中国测试行业整体效率的提升。
 
 ### MeterSphere 的功能
 
--   **测试跟踪**: 远超 TestLink 的使用体验，覆盖从编写用例到生成测试报告的完整流程；
--   **接口测试**: 集 Postman 的易用与 JMeter 的灵活于一体，接口管理、接口 Mock、多协议支持、场景自动化，你想要的全都有；
--   **性能测试**: 兼容 JMeter，支持 Kubernetes 和云环境，轻松支持高并发、分布式的性能测试；
--   **团队协作**: 用户管理、租户管理、权限管理、资源管理，无论团队规模如何，总有适合的落地方式。
+-   **测试跟踪**: 对接主流项目管理平台，测试过程全链路跟踪管理；列表脑图模式自由切换，用例编写更简单、测试报告更清晰；
+-   **接口测试**: 比 JMeter 易用，比 PostMan 强大； API 管理、Mock 服务、场景编排、多协议支持，你想要的全都有；
```
Comment (JohnNiang) [inline README.md:17]:
```suggestion
-   **接口测试**: 比 JMeter 易用，比 Postman 强大； API 管理、Mock 服务、场景编排、多协议支持，你想要的全都有；
```

**Review by wangzhen-fit2cloud — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JohnNiang) [reply README.md:17]: Please refer to https://www.postman.com/company/about-postman/.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

**Comment by JohnNiang** [comment]
Hi @wangzhen-fit2cloud ，这里有一个 check 始终失败，可以帮忙检查一下么？

- https://github.com/metersphere/metersphere/runs/6206525439?check_suite_focus=true
- https://github.com/metersphere/metersphere/runs/6206807160?check_suite_focus=true

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Comment by wangzhen-fit2cloud** [comment]
> Hi @wangzhen-fit2cloud ，这里有一个 check 始终失败，可以帮忙检查一下么？
> 
> * https://github.com/metersphere/metersphere/runs/6206525439?check_suite_focus=true
> * https://github.com/metersphere/metersphere/runs/6206807160?check_suite_focus=true

working on this

**Review by shiziyuan9527 — APPROVED** [review]

**Review by liuruibin — APPROVED** [review]

**Pull request closed without merging**